website/content/docs/integrations/vault-integration.mdx

```diff
@@ -24,21 +22,148 @@ install Vault separately from Nomad. Nomad does not run Vault for you.
+}
+```
+
+Using that configuration file, create the Vault auth method.
+
+```shell-session
+$ vault write auth/jwt/config @vault-auth-method.json
+```
+
+Next you'll write Vault policies for the role. A recommended approach is to use
+the mapping of Nomad workload claims to Vault entities you configured above to
+grant access to paths that match the claims. For example, for a Nomad job "web"
+in namespace "prod", you can grant the job automatic access to secrets at the
+Vault path `secret/data/prod/web`.
+
+To write a policy like this, first you'll need the unique Accessor of the JWT auth
+backend you created earlier.
+
+```shell-session
+$ vault auth list
+Path      Type     Accessor               Description                Version
+----      ----     --------               -----------                -------
+jwt/      jwt      auth_jwt_3a9350fe      n/a                        n/a
+```
+
+Next, write a Vault policy file using that accessor. Note that here
+`auth_jwt_3a9350fe` is the accessor shown above and you should replace it with
+the results from your own Vault cluster.
+
+```json
+path "secret/data/{{identity.entity.aliases.auth_jwt_3a9350fe.metadata.nomad_namespace}}/{{identity.entity.aliases.auth_jwt_3a9350fe.metadata.nomad_job_id}}/*" {
+  capabilities = ["read"]
+}
+
+path "secret/data/{{identity.entity.aliases.auth_jwt_3a9350fe.metadata.nomad_namespace}}/{{identity.entity.aliases.auth_jwt_3a9350fe.metadata.nomad_job_id}}" {
+  capabilities = ["read"]
+}
+
+path "secret/metadata/{{identity.entity.aliases.auth_jwt_3a9350fe.metadata.nomad_namespace}}/*" {
+  capabilities = ["list"]
+}
 
-To use the Vault integration, Nomad servers must be provided a Vault token. This
-token can either be a root token or a periodic token with permissions to create
-from a token role. The root token is the easiest way to get started, but we
-recommend a token role based token for production installations. Nomad servers
-will renew the token automatically. **Note that the Nomad clients do not need to
-be provided with a Vault token.**
+path "secret/metadata/*" {
+  capabilities = ["list"]
+}
+```
+
+Using that policy configuration file, create the Vault policy.
+
+```shell-session
+$ vault policy write nomad-workloads @vault-policy.json
+```
+
+You can write additional Vault policies that give Nomad workloads using the
+default role access to other secrets.
+
+## Authentication Without Workload Identity (Legacy)
+
+To use the legacy Vault integration, Nomad servers must be provided a Vault
+token. This token can either be a root token or a periodic token with
+permissions to create from a token role. The root token is the easiest way to
+get started, but we recommend a token role based token for production
+installations. Nomad servers will renew the token automatically. **Note that the
+Nomad clients do not need to be provided with a Vault token.**
+
+This legacy workflow will be removed in Nomad 1.9. Before upgrading to Nomad 1.9
```

Comment:
Should this paragraph perhaps be emphasized somehow? An "alert box" of some sort?

Reply:
I think we can do that. I'll add it here and to the Consul docs if it's not there already.